src/software/new_geom/util/closest_point.h

```diff
@@ -12,3 +13,15 @@
  */
 Point closestPointOnLine(const Point &p, const Line &l);
 Point closestPointOnLine(const Line &l, const Point &p);
+
+/**
+ * Finds the Point on line segment closest to point.
+ *
+ * @param p the point.
+ *
+ * @param segment the line segment.
```

Comment:
please no space between `@param`s

Reply:
Done.

Reply:
Mat and I had an offline discussion about this and didn't come to any conclusion about which is preferred (other than short comments should _probably_ be no-space.

Maybe we should have a discussion about this after alumni night